CHANGELOG_PENDING.md

```diff
@@ -4,5 +4,6 @@
 
 - Fixed refresh breaking schedules bug [#257](https://github.com/pulumi/pulumi-pulumiservice/issues/257)
 - Adjusting provider to work with updated model for DockerImage [#262](https://github.com/pulumi/pulumi-pulumiservice/issues/262)
+- Added a `CONTRIBUTING.md` guide/instructions file
```

Comment:
I think we can leave this out of the changelog since it doesn't affect the release itself.